airflow/providers/amazon/aws/operators/s3_file_transform.py

```diff
@@ -19,7 +19,7 @@
 
 import warnings
 
-from airflow.providers.amazon.aws.operators.s3 import S3FileTransformOperator  # noqa
+from airflow.providers.amazon.aws.operators.s3 import S3FileTransformOperator  # type: ignore # noqa
```

Comment:
This is caused by [s3.pyi](https://github.com/apache/airflow/blob/main/airflow/providers/amazon/aws/operators/s3.pyi) file